### ISSUE TITLE
pipes-safe build failure with ghc 8.4 (transformers-0.5.5.0)

### ISSUE THREAD

**Issue by DanBurton** [Gabriel439/Haskell-Pipes-Safe-Library#33]
```
Building library for pipes-safe-2.2.8..
[1 of 2] Compiling Pipes.Safe       ( src/Pipes/Safe.hs, .stack-work/dist/x86_64-osx/Cabal-2.2.0.0/build/Pipes/Safe.o )

/Users/dan/scratch/pipes-safe-2.2.8/src/Pipes/Safe.hs:97:7: error:
    Module ‘Control.Monad.Catch’ does not export ‘ExitCase(..)’
   |
97 |     , ExitCase(..)
   |       ^^^^^^^^^^^^
```

**Comment by Gabriel439** [comment]
Whoops!  Thanks for catching that.  I need to guard that additional imported value with CPP

**Comment by Gabriel439** [comment]
Fix uploaded to Hackage as `pipes-safe-2.2.9`
